cells-rails.gemspec

```diff
@@ -14,6 +14,12 @@ Gem::Specification.new do |spec|
   spec.summary = 'Convenient Rails support for Cells.'
   spec.homepage = 'https://trailblazer.to'
 
+  spec.metadata['homepage_uri'] = spec.homepage
+  spec.metadata['source_code_uri'] = 'https://github.com/trailblazer/cells-rails'
+  spec.metadata['bug_tracker_uri'] = "#{spec.metadata['source_code_uri']}/issues"
+  spec.metadata['changelog_uri'] = "#{spec.metadata['source_code_uri']}/blob/HEAD/CHANGES.md"
+  spec.metadata['documentation_uri'] = "https://www.rubydoc.info/gems/cells-rails/#{spec.version}"
```

Comment:
@apotonick do these look right? Should the `documentation_uri` point to https://trailblazer.to/2.1/docs/cells like the `cells` gem?

Reply:
Good point, mate! We should point here: https://trailblazer.to/2.1/docs/cells#cells-rails :green_heart:

Reply:
Updated in [`d991fdd`](https://github.com/trailblazer/cells-rails/pull/70/commits/d991fdddf9fef5b09e7fece0cdd926c1f6b4c1a9).